docs/source/faq.rst

```diff
@@ -0,0 +1,154 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can MOTECH run on Windows?
+--------------------------
+MOTECH is available and can be deployed on Windows, but it is not actively supported.
```

Comment:
but Windows is not actively supported as a platform

(noticed that this currently implies that Motech is not actively supported)